apps/site/pages/docs/component-customization/override-component.mdx

```diff
@@ -20,8 +20,9 @@ For example, you might want to change the behavior of a Price component to meet
 In this guide, you'll learn how to customize native section components by overriding them or adding extra props.
 
 
-**Overriding a component leads you to miss out on performance updates and bug fixes provided by VTEX for that component.** We highly recommend you to pass additional prop and using our theming tools to achieve the desired behavior
-without missing out on updates and bug fixes from VTEX. Check out the available props for each component on [Components UI](https://evergreen.faststore.dev/components).
+<Callout type="warning" emoji="⚠️">
+  **By overriding a component, you lose all the integration developed on [@faststore/core](docs/getting-started/project-structure#faststorecore) for that component.** *We strongly recommend that you pass additional props and use our [theming tools](/docs/customizing)* to achieve the desired behavior without compromising the existing integration implemented in `@faststore/core`. You can find the available props for each component on the [Components UI](https://evergreen.faststore.dev/components).
```

Comment:
@renatamottam, I saw below that you requested this. Can you give an example of a component-level integration we make? I'm trying to understand the seriousness of this warning.

Reply:
Sure! The [Shipping Simulation component from core](https://github.com/vtex/faststore/blob/main/packages/core/src/components/ui/ShippingSimulation/ShippingSimulation.tsx) is fully integrated with our store (api, sdk session...) and it's created using the [Shipping Simulation structure from fs/components](https://github.com/vtex/faststore/blob/main/packages/components/src/organisms/ShippingSimulation/ShippingSimulation.tsx).

If the dev decides to override the component, we provide only the `Shipping Simulation structure` (as for today). So to create a new component for the same purpose, she would have to rebuild all the integration code.